code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1760,3 +1760,67 @@
 		H.vomit(10)
 	..()
 	return TRUE // update health at end of tick
+
+/datum/reagent/medicine/neurostim
+	name = "neuro stimulant"
+	description = "A complex mixture of neuro activating chemicals and a fresh dose of a specific being DNA. the DNA seems to be that of a goat. a highly unethical mixture likely made from stem cells and other nasty bits, THERE IS ABSOLUTELY FEV IN THIS."
+	color = "#1AF708"
+	metabolization_rate = 1.0 * REAGENTS_METABOLISM
+	ghoulfriendly = TRUE
+
+/datum/reagent/medicine/neurostim/on_mob_life(mob/living/carbon/M)
+	ADD_TRAIT(M, TRAIT_SURGERY_HIGH, src)
+	ADD_TRAIT(M, TRAIT_UNETHICAL_PRACTITIONER, src)
+	ADD_TRAIT(M, TRAIT_UNETHICAL_PRACTITIONER_BRAINWASHING, src)
+	ADD_TRAIT(M, TRAIT_CYBERNETICIST, src)
+	ADD_TRAIT(M, TRAIT_CYBERNETICIST_EXPERT, src)
+	ADD_TRAIT(M, TRAIT_CHEMWHIZ, src)
+	ADD_TRAIT(M, TRAIT_MEDICALEXPERT, src)
+	ADD_TRAIT(M, TRAIT_MEDICALGRADUATE, src)
+	M.Jitter(2)
+	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, -5)
+	M.mind.teach_crafting_recipe(/datum/crafting_recipe/melee/forged/surgripper)
+	SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "jet euphoria", /datum/mood_event/jet_euphoria, name)
+	..()
+	return TRUE
+
+/datum/reagent/medicine/neurostim_na
+	name = "neuro stimulant"
+	description = "A complex mixture of neuro activating chemicals and a fresh dose of a specific being DNA. the DNA seems to be that of a goat. a highly unethical mixture likely made from stem cells and other nasty bits, THERE IS ABSOLUTELY FEV IN THIS. this syrum seems...different."
+	color = "#1AF708"
+	metabolization_rate = 1.0 * REAGENTS_METABOLISM
```

Comment:
This metabolizes at .2u/second, because of how `REAGENTS_METABOLISM` works. At 5 units, that means it lasts 13 ticks. That means it deals an average of _100 brain damage_ when used, which is classified as 'severe' brain damage. The maximum brain damage it could deal is 1040. That's on top of the stun for 40 seconds, which at 10% over 13 ticks is likely to trigger at least once.

This means you have a solid chance of stunning and killing someone within 26 seconds, which is terribly unbalanced and even with the excuse of "I don't do PVP" is just not engaging roleplay; you're forcing someone to sit through anywhere from 40 to 66 seconds of a black screen and suffer severe brain damage, which has potentially round-ruining effects.

Reply:
I will lwoer it but you or under the impression this will be used stabbing random people...it will likely be made aware it can fuck them up. OOC they know they are spending the round with me ICly teaching them shit...and guess what some people I know dont mind going thru the 30 min surgery if there brains fucked up for the purpose of RP of me fixing the fact there brain couldnt handle the stim...they have the skills there fixed in a few minutes max afterwards. this isnt a go stab some random legion pen